osu.Server.Spectator.Tests/Multiplayer/TestMultiplayerHub.cs

```diff
@@ -15,7 +15,7 @@ public TestMultiplayerHub(IDistributedCache cache, EntityStore<ServerMultiplayer
         {
         }
 
-        public ItemUsage<ServerMultiplayerRoom> GetRoom(long roomId) => Rooms.GetForUse(roomId).Result;
+        public new ItemUsage<ServerMultiplayerRoom> GetRoom(long roomId) => Rooms.GetForUse(roomId).Result;
```

Comment:
Can this be renamed to `GetRoomSynchronous` to avoid any confusion? `new` hiding always has me on edge.

Reply:
I'd just removed this method since it's available in `MultiplayerHub` itself. Touches many files but I guess it's a simple change.